tests/ert/unit_tests/config/test_queue_config.py

```diff
@@ -154,10 +157,14 @@ def test_realization_memory_unit_support(memory_spec: str, expected_bytes: int):
 @pytest.mark.parametrize(
     "invalid_memory_spec, error_message",
     [
-        ("-1", "Negative memory does not make sense in -1"),
+        ("-1", "Negative memory does not make sense"),
+        ("'      -2'", "Negative memory does not make sense"),
         ("-1b", "Negative memory does not make sense in -1b"),
-        ("b", "Could not understand byte unit in b"),
-        ("4ub", "Could not understand byte unit in 4ub"),
+        ("b", "Invalid memory string"),
+        ("'kljh3 k34f15gg.  asd '", "Invalid memory string"),
+        ("'kljh3 1gb'", "Invalid memory string"),
+        ("' 2gb 3k 1gb'", "Invalid memory string"),
+        ("4ub", "Unknown memory unit"),
```

Comment:
Is it an idea to let hypothesis generate random strings (you probably have to tell it to make millions) and look over what strings it was able to make that passes the validation (by not raising exceptions)? (not sure if it is worth the effort though)